src/app/App.module.scss

```diff
@@ -1,36 +1,52 @@
 @import "../common/colors";
```

Comment:
CSS changes in this file were made to support auto-sizing of the legacy iframe, they shouldn't affect any other layout.